xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/dto/account/BitstampRippleDepositAddress.java

```diff
@@ -44,4 +52,23 @@ public String toString() {
         ? addressAndDt
         : String.format("RippleAddress[%s, dt=%s]", address, destinationTag));
   }
+
+  static class BitstampDepositAddressDeserializer extends JsonDeserializer<BitstampDepositAddress> {
```

Comment:
This is done same way as BitcoinDepositAddress. I just took the code from there and extended it so it supports destination tag for ripple addresses. Otherwise separate method for ripple address would have to used. Now requestDepositAddressObject() will work for all currencies, for  Ripple it returns extended version of BitstampDepositAddress which has destination tag.

Reply:
ok, I'll try to fix error handlingi n this module in a separate PR